src/kdc/kdc_util.c

```diff
@@ -1698,6 +1686,8 @@ kdc_process_s4u2proxy_req(kdc_realm_t *kdc_active_realm, unsigned int flags,
     krb5_error_code errcode;
     krb5_boolean support_rbcd;
 
+    *status = "NOT_ALLOWED_TO_DELEGATE";
```

Comment:
This assignment happens on success as well as failure.  So if a later call in process_tgs_req() fails without setting the status string, we will use "NOT_ALLOWED_TO_DELEGATE" instead of "UKNOWN_REASON".  I think it's better to keep setting a status string on each error exit path.

Reply:
Fixed now.